rosie/src/main/java/com/karumi/rosie/domain/usecase/UseCaseHandler.java

```diff
@@ -29,9 +33,13 @@
   private final TaskScheduler taskScheduler;
   private final ErrorHandler errorHandler;
 
+  // This list is used to retain error adapters in a non-weak reference associated with this use case handler
+  private List<OnErrorCallbackToErrorHandlerAdapter> errorAdapters;
```

Comment:
can you initialize the collection inline and add final modifier? we are doing this one in all code only for style propuses.